getting-started/your-first-feature/0_intro.md

```diff
@@ -0,0 +1,21 @@
+This is the last in a series of three tutorials to get you up and running with full-stack DAML development. We do this through the example of a simple social networking application. The three tutorials cover:
+
+1. How to build and run the application ([Your First Feature](https://docs.daml.com/getting-started/index.html))
+1. The design of its different components ([Application Architecture](https://docs.daml.com/getting-started/app-architecture.html))
+1. How to write a new feature for the app (this tutorial)
+
+In this scenario we'll dive into implementing a new feature for our social network app. This will give us a better idea how to develop DAML applications using our template.
+
+At the moment, our app lets us follow users in the network, but we have no way to communicate with them! Let’s fix that by adding a direct messaging feature. This will allow users that follow each other send messages, repsecting authorization and privacy. This means:
+
+- You cannot send a message to someone unless they have given you the authority by following you back.
+- You cannot see a message unless you sent it or it was sent to you.
+- We will see that DAML lets us implement these guarantees in a direct and intuitive way.
+
+There are three parts to building and running the messaging feature:
+
+1. Adding the necessary changes to the DAML model
+2. Making the corresponding changes in the UI
+3. Running the app with the new feature.
```

Comment:
Either you put dots at the end of the line everywhere, or nowhere.

Reply:
Dots are everywhere now.